integration/spring-data/testsuite/webmvc/pom.xml

```diff
@@ -408,6 +408,9 @@
                                 <jdbc.password>oracle</jdbc.password>
                                 <jdbc.driver>oracle.jdbc.driver.OracleDriver</jdbc.driver>
                                 <activeProfiles>${spring.activeProfiles}</activeProfiles>
+                                <!-- Dumb, dumber, Oracle -->
```

Comment:
You also need `<jdbc.url>jdbc:oracle:thin:@localhost:1521:XE</jdbc.url>` and `<jdbc.password>Oracle18</jdbc.password>`